src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -311,6 +327,33 @@ export const useBalanceChartData: UseBalanceChartData = args => {
   const balances = useAppSelector(state =>
     selectPortfolioCryptoBalancesByAccountIdAboveThreshold(state, accountId)
   )
+
+  // Get total delegation
+  // TODO(ryankk): consolidate accountSpecifiers creation to be the same everywhere
+  const cosmosCaip2: CAIP2 = caip2.toCAIP2({
```

Comment:
i hate that we're doing this stuff in what is essentially the view layer.

we really need to settle on a single source of the truth for supported chains

getting it from here is *better*, but still bad, and this should come from chain adapters

https://github.com/shapeshift/web/blob/develop/src/state/slices/portfolioSlice/utils.ts#L27

Reply:
I would argue using toCaip2 is just as clear as importing it from portfolio slice